packages/m2m-gateway/src/clients/clientsProvider.ts

```diff
@@ -157,5 +162,9 @@ export function getInteropBeClients(): PagoPAInteropBeClients {
       eserviceTemplateApi.createProcessApiClient,
       config.eserviceTemplateProcessUrl
     ),
+    eventManagerClient: createZodiosClientEnhancedWithMetadata(
+      m2mEventApi.createM2mEventsApiClient,
+      config.eventManagerUrl
+    ),
```

Comment:
Same, no need for metadata patch here (make sure you perform again a manual test after this to check that everything still works)

```suggestion
    eventManagerClient: m2mEventApi.createM2mEventsApiClient(
      config.eventManagerUrl
    ),
```